src/components/ui/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Accordion, AccordionDetails, AccordionSummary } from './index';
+
+export default {
+  title: 'Ui/Accordion',
+  component: Accordion,
+  argTypes: {},
+} as ComponentMeta<typeof Accordion>;
+
+const Template: ComponentStory<typeof Accordion> = (args) => (
+  <Accordion {...args}>
+    <AccordionSummary>Foo</AccordionSummary>
+
+    <AccordionDetails>Bar</AccordionDetails>
+  </Accordion>
+);
+
+export const Expanded = Template.bind({});
+Expanded.args = {
+  isExpanded: true,
+};
+
+export const NotExpanded = Template.bind({});
```

Comment:
I think this test case is not needed since we have controls and can expand it
![image](https://user-images.githubusercontent.com/86728658/152315931-18bc5f7e-1f37-4c48-acb8-bd4ddcd37845.png)